app/messages/summaries.email.html.erb

```diff
@@ -11,17 +11,19 @@
 <% end %>
 
 <% define_content :footer_link do %>
+<!--
```

Comment:
My guess is that this is the link we want to keep and the ```:footer_link``` is the duplicate one that takes you to the root URL.  Am I wrong?  In an actual email, one link is to https://portal.beyond.org and the other is to https://portal.beyondz.org/profile/communication.  Which one is this commenting out?